modules/social_features/social_content_report/config/schema/social_content_report.schema.yml

```
@@ -0,0 +1,16 @@
+# Schema for the configuration files of the Social Content Report module.
+social_content_report.settings:
+  type: config_object
+  label: 'Social Content Moderation settings'
+  mapping:
+    unpublish_threshold:
+      type: boolean
```

```suggestion
      type: integer
```
This threshold should be an integer. In the future we want to be able to only unpublish content at 3 reports. Changing this configuration now is easier than later and a boolean 0/1 value still has the desired effect (0 = don't unpublish, 1 = unpublish immediately).